test/models/partner_relationship_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Since we are using RSpec for test in this application, we should instead add basic model specs for Partner Relationships like this one:  https://github.com/CZagrobelny/new_sanctuary_asylum/blob/master/spec/models/spousal_relationship_spec.rb
We should also add a context for 'Partner' in the Family Member Constructor Spec, following the example of the context for Spouse in the spec here: https://github.com/CZagrobelny/new_sanctuary_asylum/blob/master/spec/models/family_member_constructor_spec.rb#L16